.github/ci.sh

```diff
@@ -66,15 +47,7 @@ build() {
   if [[ "$ENABLE_HPC" == "true" ]]; then
     cat cabal.project.ci-hpc >> cabal.project.local
   fi
-  if ! retry cabal v2-build "$@" "${pkgs[@]}"; then
-    if [[ "$RUNNER_OS" == "macOS" ]]; then
```

Comment:
Might be worth leaving a comment here in case this MacOS dylib issue crops up again.